lib/workers/repository/errors-warnings.ts

```diff
@@ -69,14 +70,12 @@ export function getDepWarningsPR(
   warningText = emojify(
     `\n---\n\n### :warning: Dependency Lookup Warnings :warning:\n\n`
   );
-  warningText += `Please correct - or verify that you can safely ignore - these lookup failures before you merge this PR.\n\n`;
```

Comment:
Isn't this line from the onboarding PR? Will it be replaced with the new messages? Is this intentional?

Reply:
we wanted to keep the warning text in the PR body as simple as possible.
do you want to add a "safely ignore" message